components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.common/src/main/java/org/wso2/carbon/identity/api/server/configs/common/Constants.java

```diff
@@ -46,13 +62,34 @@ private Constants() {
         ERROR_CODE_ERROR_RETRIEVING_AUTHENTICATOR("65003",
                 "Unable to retrieve authenticator.",
                 "Server encountered an error while retrieving the authenticator for the identifier %s."),
-        ERROR_CODE_AUTHENTICATOR_NOT_FOUND("60002", "Resource not found.",
+        ERROR_CODE_AUTHENTICATOR_NOT_FOUND("60002",
+                "Resource not found.",
                 "Unable to find a resource matching the provided authenticator identifier %s."),
-        ERROR_CODE_INVALID_INPUT("60003", "Invalid input.", "One of the given inputs is invalid. %s."),
-        ERROR_CODE_ERROR_UPDATING_CONFIGS("65004", "Unable to update server configs.", "Server encountered an " +
+        ERROR_CODE_INVALID_INPUT("60003",
+                "Invalid input.",
+                "One of the given inputs is invalid. %s."),
+        ERROR_CODE_ERROR_UPDATING_CONFIGS("65004",
+                "Unable to update server configs.",
+                "Server encountered an " +
                 "error while updating the server configs."),
-        ERROR_CODE_ERROR_RETRIEVING_CONFIGS("65005", "Unable to retrieve server configs.", "Server encountered an " +
-                "error while retrieving the server configs.");
+        ERROR_CODE_ERROR_RETRIEVING_CONFIGS("65005",
+                "Unable to retrieve server configs.",
+                "Server encountered an " +
+                "error while retrieving the server configs."),
```

Comment:
can't we inline 77 and 78?

Reply:
https://github.com/wso2/identity-api-server/pull/177#discussion_r456350591